mw-webapp/src/component/table/columns.tsx

```diff
@@ -24,7 +28,7 @@ const getStringArrayItem = (arrayItem: string, index: string) => {
       <div />
       :
       <div key={index}>
-        <div className={arrayItem[0] === "✓" ? styles.completed : styles.notCompleted}>
+        <div className={arrayItem[FIRST_INDEX] === "✓" ? styles.completed : styles.notCompleted}>
```

Comment:
too raugh check by first char in string, better to use flag

Reply:
Create an issue #65

Reply:
also mark it with TODO in this case, because it will be hard to find it in future

Reply:
Done